pkg/agent/secondarynetwork/init.go

```diff
@@ -55,6 +61,31 @@ func Initialize(
 		return err
 	}
 
+	var addPrimaryInterfaceToBridge bool
+	for _, phyInterface := range config.OVSBridges[0].PhysicalInterfaces {
+		if phyInterface == primaryInterfaceName {
+			addPrimaryInterfaceToBridge = true
+			break
+		}
+	}
+
+	if addPrimaryInterfaceToBridge {
```

Comment:
I think no need to differentiate primary or other interfaces. We can always move the IPs and routes on the physical interface to the bridge, which there is only one physical interface configured. It is always good to keep the IPs and routes on interfaces.